support-frontend/assets/pages/supporter-plus-landing/twoStepPages/checkoutScaffold.tsx

```diff
@@ -166,7 +167,7 @@ export function SupporterPlusCheckoutScaffold({
 
 	useEffect(() => {
 		if (paymentComplete) {
-			navigate(thankYouRoute, { replace: true });
```

Comment:
I'm interested in the `{ replace: true }` options argument being passed in the original implementation. According to React Router's [docs](https://reactrouter.com/en/main/hooks/use-navigate#optionsreplace): 

> Specifying replace: true will cause the navigation to replace the current entry in the history stack instead of adding a new one.

I think we need to look at the impact of not using this in the update, as we may need to extend `navigateWithPageView` to take these options.

Reply:
Yep, good spot. I should of routed this through straight away. Have updated call with optional parameter to do this now and defaulted to `false` without